dot/peerset/peerset.go

```diff
@@ -340,7 +340,7 @@ func (ps *PeerSet) reportPeer(change ReputationChange, peers ...peer.ID) error {
 	for _, pid := range peers {
 		rep, err := ps.peerState.addReputation(pid, change)
 		if err != nil {
-			return fmt.Errorf("cannot add reputation (%s) to peer %s: %w", change, pid, err)
+			return fmt.Errorf("cannot add reputation: %w", err)
```

Comment:
nit add the change to the error context in `addReputation`: `fmt.Errorf("%w: for peer id %s", ErrPeerDoesNotExist, peerID)`